codegen/type_converter.go

```diff
@@ -81,6 +82,12 @@ func NewTypeConverter(h PackageNameResolver) *TypeConverter {
 	}
 }
 
+func (c *TypeConverter) makeUniqIdentifier(prefix string) string {
```

Comment:
it seems you have an assumption here that `makeUniqIdentifier` always used together for key/val (or items at the same level), shall we either generate a pair of strings as it used, or leave a comment to avoid case like
```
for key1, val2 := range ....
```
?

Reply:
this is used for 5/6 different variable names. We could try to "pretty" them but im happy enough with numerically unique postfixes.